packages/lisk-p2p/test/unit/errors.ts

```diff
@@ -83,4 +87,74 @@ describe('errors', () => {
 			});
 		});
 	});
+
+	describe('#RPCGetPeersFailed', () => {
```

Comment:
Don't worry about it for now but maybe we shouldn't be so detailed with the tests at this early stage in the project? Makes it harder to change things.

Testing `cause` property might be overkill since this functionality is related to the VError object and should already be covered by unit tests inside the VError library.

Reply:
You are right that we should not make very detailed test cases at this stage, I will remove the unnecessary ones. Regarding `cause`, I'm not testing VError functionality exactly but testing if our custom error `RPCGetPeersFailed ` is able to set cause property of `VError`